src/SoftRobots.Inverse/component/constraint/BarycentricCenterEffector.inl

```diff
@@ -54,19 +56,16 @@ BarycentricCenterEffector<DataTypes>::BarycentricCenterEffector(MechanicalState*
                       "which you want to solve the effector. If unspecified, the default    \n"
                       "values are {true, true, true})."))
 
-    , d_effectorGoalPosition(initData(&d_effectorGoalPosition,"effectorGoal",
-                                      "The parameter effectorGoal allows to specifiy the desired position of the\n"
-                                      "constraint. If unspecified the default values are {0.0,0.0,0.0}" ))
-
     , d_drawBarycenter(initData(&d_drawBarycenter,false,"drawBarycenter",
                                 "If true, draw the barycenter" ))
 
     , d_barycenter(initData(&d_barycenter,"barycenter",
                             "Position of barycenter." ))
-
-    , d_delta(initData(&d_delta,"delta",
-                            "Distance to target" ))
 {
+    d_axis.setDisplayed(false);
+    d_indices.setDisplayed(false); // inherited from PositionModel but not used here
+    d_drawBarycenter.setGroup("Visualization");
```

Comment:
Normally, it is already in this group since the name starts with `draw`. See https://github.com/sofa-framework/sofa/blob/master/Sofa/framework/Core/src/sofa/core/objectmodel/Base.cpp#L160